Workflow/Sources/Workflow/Models/AnyWorkflow.swift

```diff
@@ -0,0 +1,187 @@
+//
+//  AnyWorkflow.swift
+//  
+//
+//  Created by Tyler Thompson on 11/25/20.
+//  Copyright © 2021 WWT and Tyler Thompson. All rights reserved.
+//
+
+import Foundation
+
+public class AnyWorkflow: LinkedList<FlowRepresentableMetadata> {
+    public typealias InstanceNode = LinkedList<AnyFlowRepresentable?>.Element
+    public typealias ArrayLiteralElement = AnyFlowRepresentable.Type
+    internal var instances = LinkedList<AnyFlowRepresentable?>()
+    internal var orchestrationResponder: AnyOrchestrationResponder?
+
+    public func applyOrchestrationResponder(_ orchestrationResponder: AnyOrchestrationResponder) {
+        self.orchestrationResponder = orchestrationResponder
+    }
+
+    @discardableResult public func launch(withLaunchStyle launchStyle: LaunchStyle = .default,
+                                          onFinish: ((Any?) -> Void)? = nil) -> LinkedList<AnyFlowRepresentable?>.Element? {
```

Comment:
Should all these `onFinish` blocks actually take in an `AnyWorkflow.PassedArgs` enum? Consumers can then make deliberate decisions if no data was passed forward at the end of a workflow. Right now consumers cannot distinguish between `nil` being a legitimately passed value and nothing being passed. That distinction may be very important

Reply:
Yes they should and good job doing that.